pkg/multiclustermanager/routes.go

```diff
@@ -68,6 +69,7 @@ func router(ctx context.Context, localClusterEnabled bool, tunnelAuthorizer *mcm
 
 	channelserver := channelserver.NewHandler(ctx)
 
+	supportConfigGenerator := supportconfigs.NewGeneratorHandler(scaledContext)
```

Comment:
Pretty sure scaled context is correct here, since any of the rancher pods could get the api hit, not just the leader.